pepys_import/utils/geoalchemy_utils.py

```diff
@@ -0,0 +1,6 @@
+EXTENSION_PATH = "/usr/lib/x86_64-linux-gnu/mod_spatialite.so"
+
```

Comment:
@BarisSari - could you insert an O/S specific flag please?

On my MacOS the path is `"/usr/local/lib/mod_spatialite.dylib"`

Reply:
Added.

https://github.com/debrief/pepys-import/pull/43/commits/15de49b6c54c568e99d0787e5ecba54da815d225